packages/integrations/components/coral/index.jsx

```diff
@@ -44,29 +44,6 @@ const CoralEmbed = ({
   // https://reactjs.org/docs/hooks-rules.html
   const shouldPurgeUser = useSelector(purgeSelector);
 
-  /**
```

Comment:
There's nothing more satisfying that cleaning up hacky code to handle edge-cases 🤩